packages/waku/src/router/client.ts

```diff
@@ -261,9 +263,16 @@ export function Link({
     }
   }, [unstable_prefetchOnView, router, to]);
   const onClick = (event: MouseEvent<HTMLAnchorElement>) => {
-    event.preventDefault();
+    const isAltClick =
+      unstable_defaultAltClick &&
+      Boolean(
+        event.metaKey || event.ctrlKey || event.shiftKey || event.button !== 0,
+      );
     const url = new URL(to, window.location.href);
-    if (url.href !== window.location.href) {
+    if (!isAltClick) {
+      event.preventDefault();
+    }
```

Comment:
Why can it be this?
```js
if (isAltClick) {
  return;
}
```

Reply:
That skips the onClick prop. What do you think about firing the onClick prop first? React Router does that and then doesn't run the internal handler if the onClick calls preventDefault().
https://github.com/remix-run/react-router/blob/react-router%407.2.0/packages/react-router/lib/dom/lib.tsx#L628-L635

Reply:
Oh, that looks great. Any drawbacks?

Reply:
Not that I can think of. It looks good to me. I'm traveling for the next few days so I probably won't push another update until the beginning of next week.

Reply:
ping